RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingUtils/formatHelpers.js

```diff
@@ -25,13 +25,39 @@ const DAYNAME = [
   customLabels.Appointment_ReBooking_WeekDayLong_Saturday
 ];
 
+const getFormattedTime = (date) => {
```

Comment:
Simply moved these up here instead since these consts should be defined first before being used in other consts